src/main/java/at/ac/fhcampuswien/fhmdb/ui/MovieCell.java

```diff
@@ -1,73 +1,44 @@
 package at.ac.fhcampuswien.fhmdb.ui;
 
 import at.ac.fhcampuswien.fhmdb.models.Movie;
-import javafx.geometry.Insets;
-import javafx.scene.control.Label;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Node;
 import javafx.scene.control.ListCell;
-import javafx.scene.layout.Background;
-import javafx.scene.layout.BackgroundFill;
-import javafx.scene.layout.VBox;
-import javafx.scene.paint.Color;
 
-import java.util.List;
+import java.io.IOException;
 
 public class MovieCell extends ListCell<Movie> {
-    private static final String TEXT_WHITE = "text-white";
-    private static final String TEXT_YELLOW = "text-yellow";
-    private final Label title = new Label();
-    private final Label detail = new Label();
-    private final Label genre = new Label();
-    private final Label movieDetails = new Label();
-    private final Label producingCast = new Label();
-    private final Label directors = new Label();
-    private final Label mainCast = new Label();
-    private final VBox layout = new VBox(title, detail, genre, movieDetails, producingCast, mainCast);
 
     @Override
     protected void updateItem(Movie movie, boolean empty) {
+        Node graphic;
+        MovieCellController controller;
+
+        try {
+            FXMLLoader loader = new FXMLLoader(MovieCell.class.getResource("movie-cell.fxml"));
+            graphic = loader.load();
+            controller = loader.getController();
```

Comment:
I wouldn't keep these lines in the update method, because this is called several times for each item in the list when you update the view and the loader and the controller for instance won't change.
Try moving them in the constructor or in a initialize block

Reply:
Moving it into the constructor or initialize block creates the bug where the items toggle the details randomly on and off when sorting.